File: java/TiketBioskop.java
public class TiketBioskop extends TiketFilm {
    private String studio;
    private String jamTayang;
    private String tanggalTayang;
    private String jenisStudio;

    public TiketBioskop(
        String idTiket,
        String nomorKursi,
        int harga,
        String statusTiket,
        String judulFilm,
        String genre,
        int durasi,
        int ratingUsia,
        String studio,
        String jamTayang,
        String tanggalTayang,
        String jenisStudio
    ) {
        super(
            idTiket,
            nomorKursi,
            harga,
            statusTiket,
            judulFilm,
            genre,
            durasi,
            ratingUsia
        );

        this.studio = studio;
        this.jamTayang = jamTayang;
        this.tanggalTayang = tanggalTayang;
        this.jenisStudio = jenisStudio;
    }

    public String getStudio() {
        return studio;
    }

    public String getJamTayang() {
        return jamTayang;
    }

    public String getTanggalTayang() {
        return tanggalTayang;
    }

    public String getJenisStudio() {
        return jenisStudio;
    }

    public void setStudio(String studio) {
        this.studio = studio;
    }

    public void setJamTayang(String jamTayang) {
        this.jamTayang = jamTayang;
    }

    public void setTanggalTayang(String tanggalTayang) {
        this.tanggalTayang = tanggalTayang;
    }

    public void setJenisStudio(String jenisStudio) {
        this.jenisStudio = jenisStudio;
    }
}

File: python/TiketBioskop.py
from TiketFilm import TiketFilm


class TiketBioskop(TiketFilm):
    def __init__(
        self,
        idTiket,
        nomorKursi,
        harga,
        statusTiket,
        judulFilm,
        genre,
        durasi,
        ratingUsia,
        studio,
        jamTayang,
        tanggalTayang,
        jenisStudio
    ):
        super().__init__(
            idTiket,
            nomorKursi,
            harga,
            statusTiket,
            judulFilm,
            genre,
            durasi,
            ratingUsia
        )

        self.__studio = str(studio)
        self.__jamTayang = str(jamTayang)
        self.__tanggalTayang = str(tanggalTayang)
        self.__jenisStudio = str(jenisStudio)

    def getStudio(self):
        return self.__studio

    def getJamTayang(self):
        return self.__jamTayang

    def getTanggalTayang(self):
        return self.__tanggalTayang

    def getJenisStudio(self):
        return self.__jenisStudio

    def setStudio(self, studio):
        self.__studio = str(studio)

    def setJamTayang(self, jamTayang):
        self.__jamTayang = str(jamTayang)

    def setTanggalTayang(self, tanggalTayang):
        self.__tanggalTayang = str(tanggalTayang)

    def setJenisStudio(self, jenisStudio):
        self.__jenisStudio = str(jenisStudio)